packaging/windows/clisdk/bundle.wxs

```diff
@@ -23,12 +23,9 @@
 
     <swid:Tag Regid="microsoft.com" InstallPath="[DOTNETHOME]" />
 
-    <Variable Name="DOTNETHOME" Type="string" Value="[ProgramFiles6432Folder]dotnet" bal:Overridable="yes" />
+    <Variable Name="DOTNETHOME" Type="string" Value="[ProgramFiles6432Folder]dotnet" bal:Overridable="no" />
```

Comment:
This means developers can't change the install location of the bundle. Do you want to prevent install folder customization? If so, then we can remove a lot of this stuff and avoid the problems completely.